CONTRIBUTING.md

```diff
@@ -23,7 +23,17 @@ information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
 
-## Instructions
+## Required Tools
+
+Working with the project sources requires the following tools:
+
+1. [git](https://git-scm.com/)
```

Comment:
One think that you may want to add is that some of the tools we use during the build like golint are installed into the `$GOPATH/bin` directory, so one required thing may be to do is `PATH=$GOPATH/bin:$PATH`

Reply:
I will add a note to that effect